py3status/modules/insync.py

```diff
@@ -85,14 +87,17 @@ def insync(self):
             }
 
         color = self.py3.COLOR_DEGRADED
+        format = self.format
         if status == "Insync doesn't seem to be running. Start it first.":
             color = self.py3.COLOR_BAD
-            status = STRING_ERROR
+            format = STRING_ERROR
```

Comment:
if we change this to 
```
if status == "Insync doesn't seem to be running. Start it first.":
    self.py3.error(STRING_ERROR)
```

then I think we can merge this PR.  That way the error gets handled consistently across modules.  This is relatively new functionality so some existing modules don't yet use it.